types/p2/index.d.ts

```diff
@@ -1,21 +1,18 @@
 // Type definitions for p2.js v0.7.1
 // Project: https://github.com/schteppe/p2.js/
-// Definitions by: Clark Stevenson <https://github.com/clark-stevenson>, Janne Ramstedt <https://github.com/jramstedt>, Alex Brown <https://github.com/JellyAlex>
+// Definitions by: Clark Stevenson <https://github.com/clark-stevenson>, Janne Ramstedt <https://github.com/jramstedt>, Alex Brown <https://github.com/alexandermbrown>
```

Comment:
```suggestion
// Definitions by: Clark Stevenson <https://github.com/clark-stevenson>, Janne Ramstedt <https://github.com/jramstedt>, Alex Brown <https://github.com/JellyAlex>
```

Reply:
Please accept this suggestion, I've created separate PR to remove Alex from code owners (as requested in a comment)